code_schemes/s01e02.json

```
@@ -3,6 +3,78 @@
     "Name": "RQA S01E02",
     "Version": "0.0.0.1",
     "Codes": [
+        {
+            "CodeID": "code-9072263b",
+            "CodeType": "Normal",
+            "DisplayText": "election has caused war and bloodshed",
+            "NumericValue": 1,
+            "StringValue": "election_has_caused_war_and_bloodshed",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-6f546f0c",
+            "CodeType": "Normal",
+            "DisplayText": "insecurity",
+            "NumericValue": 2,
+            "StringValue": "insecurity",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-347d263a",
+            "CodeType": "Normal",
+            "DisplayText": "economic hardships",
+            "NumericValue": 3,
+            "StringValue": "economic_hardships",
+            "VisibleInCoda": true
+        },
+        {
+            "CodeID": "code-ac522bc9",
+            "CodeType": "Normal",
+            "DisplayText": "the election has not caused war and bloodshed",
```

consider
```suggestion
            "DisplayText": "election has not caused war and bloodshed",
```
for consistency with code 1